{
  "name": "scala.util.control.Exception$By",
  "ancestorCount": 1,
  "isStaticModule": false,
  "isInterface": false,
  "isImplClass": false,
  "isRawJSType": false,
  "encodedName": "scala_util_control_Exception$By",
  "superClass": "java_lang_Object",
  "ancestors": ["scala_util_control_Exception$By", "java_lang_Object"],
  "isExported": false,
  "methods": {
    "__init__": {},
    "by__O__O": {
      "calledMethods": {
        "scala_Function1": ["apply__O__O"]
      }
    },
    "init___Lscala_Function1": {
      "calledMethodsStatic": {
        "java_lang_Object": ["init___"]
      }
    },
    "by__O__": {
      "calledMethods": {
        "scala_util_control_Exception$By": ["by__O__O"]
      }
    }
  }
}
